src/api/resources/accounts.js

```diff
@@ -26,5 +26,17 @@ export default class AccountsResource extends APIResource {
 		this.get = apiMethod({
 			method: GET,
```

Comment:
This is the default. Do we want to be explicit here?

Reply:
I think we should be explicit as much as possible